docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task or event: `add`
 
-> Persons can have any number of tags (including 0)
+Adds a task to Lifekeeper<br>
+Format: `add TASK_NAME [d/DEADLINE] [p/PRIORITY_LEVEL] [r/REMINDER] [t/TAG]...` 
+
+> Tasks can have any number of tags (including 0)
+
+`DEADLINE` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+
+`PRIORITY_LEVEL` has to be an integer equal or larger than 1, with `1` being the top priority.
+
+`REMINDER` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+* `[TIME] before` sets reminder at the specified time before the `DEADLINE`. e.g. `0015 before` for a reminder 15 minutes before the `DEADLINE`.
+
+Examples: 
+* `add Grocery Shopping`
+* `add Assignment 1 d/Tomorrow p/1 r/Today 2000`
+* `add Assignment 2 d/Tomorrow p/1 r/0400 before`
+* `add CS2103 T7A1 d/6 Oct 2016 p/2 r/5 Oct 2016 1800 t/teamC2`
+
+Adds an event to Lifekeeper<br>
+Format: `add EVENT_NAME s/START_TIME e/END_TIME [t/TAG]...` 
+
+> Events can have any number of tags (including 0)
+
+`START_TIME` and `END_TIME` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+* `TIME` format assumes `today`
+
+Examples: 
+* `add Lunch s/1200 e/1300`
+* `add Executive Meeting s/tomorrow 0900 e/tomorrow 1200`
+* `add Concert s/tomorrow 1800 e/tomorrow 2000 t/Leisure`
+
+#### Adding a category: `addcat`
+Adds a category to Lifekeeper<br>
+Format: `add CATEGORY_NAME` 
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `addcat Work`
+* `addcat CS2103`
+* `addcat Family `
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
+#### Listing tasks : `list`
+Shows a list of tasks in Lifekeeper, if any.<br>
 Format: `list`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> All the tasks in Lifekeeper will be listed
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+#### Listing categories: `listcat`
+Shows a list of all categories in LifeKeeper.<br>
+Format: `listcat`
+
+> All the categories in Lifekeeper will be listed
+
+
+#### Finding tasks by name, tag(s) or the range of DEADLINEs: `find`
```

Comment:
again, like the add, provide a generic command format first,
then explain the variations as below.